tests/lib/nested.sh

```diff
@@ -24,7 +24,8 @@ NESTED_FAKESTORE_SNAP_DECL_PC_GADGET="${NESTED_FAKESTORE_SNAP_DECL_PC_GADGET:-}"
 NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL="${NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL:-}"
 NESTED_UBUNTU_IMAGE_PRESEED_KEY="${NESTED_UBUNTU_IMAGE_PRESEED_KEY:-}"
 
-NESTED_PHYSICAL_4K_SECTOR_SIZE="${NESTED_PHYSICAL_4K_SECTOR_SIZE:-}"
+: "${NESTED_DISK_PHYSICAL_BLOCK_SIZE:=512}"
```

Comment:
(nitpick) this is nice but I would prefer if we keep things uniform, i.e. keep using the exiting pattern:
```bash
diff --git a/tests/lib/nested.sh b/tests/lib/nested.sh
index 8e3c9bfb5d..4d8ef6c162 100644
--- a/tests/lib/nested.sh
+++ b/tests/lib/nested.sh
@@ -24,8 +24,8 @@ NESTED_FAKESTORE_SNAP_DECL_PC_GADGET="${NESTED_FAKESTORE_SNAP_DECL_PC_GADGET:-}"
 NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL="${NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL:-}"
 NESTED_UBUNTU_IMAGE_PRESEED_KEY="${NESTED_UBUNTU_IMAGE_PRESEED_KEY:-}"
 
-: "${NESTED_DISK_PHYSICAL_BLOCK_SIZE:=512}"
-: "${NESTED_DISK_LOGICAL_BLOCK_SIZE:=512}"
+NESTED_DISK_PHYSICAL_BLOCK_SIZE="${NESTED_DISK_PHYSICAL_BLOCK_SIZE:-512}"
+NESTED_DISK_LOGICAL_BLOCK_SIZE="${NESTED_DISK_LOGICAL_BLOCK_SIZE:-512}"
 
 nested_wait_for_ssh() {
     # TODO:UC20: the retry count should be lowered to something more reasonable.
```
I think moving everything to the `:` is also fine but maybe in a followup and then for all of the vars where it makes sense in this script?